assets/src/dashboard/app/api/useStoryApi.js

```diff
@@ -251,12 +252,15 @@ const useStoryApi = (dataAdapter, { editStoryURL, storyApi, encodeMarkup }) => {
           story: {
             status: status || 'auto-draft',
             title: title,
-            author: author || 1,
+            author: { id: author || 1, name: '' },
             slug: title,
             date: created || Date.now().toString(),
             modified: modified || Date.now().toString(),
             featuredMedia: {
               id: 0,
+              url: '',
+              width: 1,
+              height: 1,
```

Comment:
I suspect that the required shape for getStoryMarkup was updated and this lagged behind. Just adding in the things the api wants that we don't ever actually need to have a preview dialog so that the preview will work.